### ISSUE TITLE
Upgrade containers to use Rocky Linux 9 and AWS SDK CPP 1.9.365

### ISSUE THREAD

**Issue by adamhgriffith-uofu** [slateci/docker-images#87]
Update slate-client-server image to use Rocky Linux 9 and use the latest aws sdk cpp (1.9.365).  Work ongoing in https://github.com/slateci/docker-images/tree/rocky_update and https://github.com/slateci/slate-client-server/tree/opentelemetry

**Comment by adamhgriffith-uofu** [comment]
Work in https://github.com/slateci/docker-images has been completed as of #88.

**Comment by adamhgriffith-uofu** [comment]
@sthapa, in the [opentelemetry branch](https://github.com/slateci/slate-client-server/tree/opentelemetry) I have updated references for the `slate-client-server` utility image from `1.X.X` to `2.0.0` as of https://github.com/slateci/slate-client-server/commit/4ea34ceebc25c160946900faa5573a7b2cd1f0e6.

However when trying to build this branch in the utility image I get:
```shell
[root@1fffd2d93bed build]# cmake3 .. -DBUILD_CLIENT=False -DBUILD_SERVER=True -DBUILD_SERVER_TESTS=False -DSTATIC_CLIENT=False
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 11.2.1
-- The CXX compiler identification is GNU 11.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Will build server
-- Found ZLIB: /usr/lib64/libz.so (found version "1.2.11") 
CMake Warning (dev) at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (PkgConfig)
  does not match the name of the calling package (libcrypto).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPkgConfig.cmake:70 (find_package_handle_standard_args)
  cmake/Packages/Findlibcrypto.cmake:3 (include)
  CMakeLists.txt:68 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PkgConfig: /usr/bin/pkg-config (found version "1.7.3") 
-- Found libcrypto:  (found version "3.0.1")
CMake Warning (dev) at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (PkgConfig)
  does not match the name of the calling package (ssl).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPkgConfig.cmake:70 (find_package_handle_standard_args)
  cmake/Packages/Findssl.cmake:3 (include)
  CMakeLists.txt:69 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found ssl:  (found version "3.0.1")
-- Found CURL: /usr/lib64/libcurl.so (found version "7.76.1")  
-- Found AWS SDK for C++, Version: 1.9.365, Install Root:/usr/local, Platform Prefix:, Platform Dependent Libraries: pthread;crypto;ssl;z;curl
-- Components specified for AWSSDK: core, application will be depending on libs: aws-cpp-sdk-core;aws-crt-cpp;aws-c-auth;aws-c-cal;aws-c-common;aws-c-compression;aws-c-event-stream;aws-c-http;aws-c-io;aws-c-mqtt;aws-c-s3;aws-checksums;aws-c-sdkutils;pthread;crypto;ssl;z;curl
-- Try finding aws-cpp-sdk-core
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found crypto: /usr/lib64/libcrypto.so  
-- LibCrypto Include Dir: /usr/include
-- LibCrypto Shared Lib:  /usr/lib64/libcrypto.so
-- LibCrypto Static Lib:  crypto_STATIC_LIBRARY-NOTFOUND
-- Found OpenSSL: /usr/lib64/libcrypto.so (found version "3.0.1")  
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- S2N found target: AWS::crypto
-- crypto Include Dir: /usr/include
-- Found aws-cpp-sdk-core
-- Found Boost: /usr/include (found version "1.75.0") found components: date_time system 
CMake Warning (dev) at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (PkgConfig)
  does not match the name of the calling package (yaml-cpp).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPkgConfig.cmake:70 (find_package_handle_standard_args)
  cmake/Packages/Findyaml-cpp.cmake:3 (include)
  CMakeLists.txt:75 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found yaml-cpp: /usr (found version "0.6.3")
CMake Error at CMakeLists.txt:76 (find_package):
  Could not find a package configuration file provided by "opentelemetry-cpp"
  with any of the following names:

    opentelemetry-cppConfig.cmake
    opentelemetry-cpp-config.cmake

  Add the installation prefix of "opentelemetry-cpp" to CMAKE_PREFIX_PATH or
  set "opentelemetry-cpp_DIR" to a directory containing one of the above
  files.  If "opentelemetry-cpp" provides a separate development package or
  SDK, be sure it has been installed.


-- Configuring incomplete, errors occurred!
See also "/slate/build/CMakeFiles/CMakeOutput.log".
```
You may already have fixed this locally or I am missing a parameter somewhere.

**Comment by adamhgriffith-uofu** [comment]
*develop* and the buildtime `./resources/docker/Dockerfile` have been updated to work with Rocky9 and the static build of the AWS C++ SDK 1.9.365.
* I did have to deploy `hub.opensciencegrid.org/slate/slate-client-server:2.0.1` so @sthapa will need to update his base image reference in the CLion Docker file.
* I confirmed that on *develop* the current API server source does compile under these new conditions.
* At this point with the upcoming merge of the *opentelemetry* branch we should be good to go?
